### PR TITLE
Some refactoring

### PR DESCRIPTION
**Pull request by Salmela**
Some small refactoring. ;-)

I will continue this work after about two weeks...

### CODE REVIEW

File: src/main/java/fi/helsinki/cs/tmc/cli/command/InfoCommand.java [hunk 1]
```diff
@@ -241,11 +242,7 @@ private void printExercise(Exercise exercise) {
     }
 
     private String getDeadline(Exercise exercise) {
-        String deadline = exercise.getDeadline();
-        if (deadline == null) {
-            return "not available";
```
Comment (Salmela) [inline src/main/java/fi/helsinki/cs/tmc/cli/command/InfoCommand.java:246]: I accidentally removed this if block

File: src/main/java/fi/helsinki/cs/tmc/cli/command/InfoCommand.java [hunk 2]
```diff
@@ -152,58 +165,33 @@ private void printCourseDetails() {
...
-            printExerciseShort();
+            printExerciseShort(exercise);
         }
     }
 
-    private void printExerciseShort() {
+    private void printExerciseShort(Exercise exercise) {
         io.println("Exercise: " + exercise.getName());
-        io.println("Deadline: " + getDeadline(exercise));
+        io.println("Deadline: " + exercise.getDeadlineDate());
```
Comment (Salmela) [inline src/main/java/fi/helsinki/cs/tmc/cli/command/InfoCommand.java:178]: TODO
Check if DeadlineDate is null and print "none" if it is null.
And create unit test for it.

**Comment by Salmela** [comment]
I don't mind if this pull request is merged to master. I will then just create new branch after my trip.

**Comment by jclc** [comment]
There is no rush to merge this massive refactoring yet, since it's not currently actively developed.

**Pull request merged**